apel/db/backends/mysql.py

```diff
@@ -404,6 +404,51 @@ def summarise_cloud(self):
                 self.db.rollback()
             raise
 
+    def clean_stale_cloud_summaries(self, summariser_start_time, threshold):
+        """Remove stale cloud summaries newer than the threshold."""
+        threshold_timedelta = datetime.timedelta(days=threshold)
+        threshold_time = summariser_start_time - threshold_timedelta
+        threshold_month = threshold_time.month
+        threshold_year = threshold_time.year
+
+        try:
+            # prevent MySQLdb from raising
+            # 'MySQL server has gone' exception
+            self._mysql_reconnect()
+
+            cursor = self.db.cursor()
+            # Delete any summaries that:
+            # - Are older than the start time of this summariser run (i.e. are
+            #   stale)
+            # AND
+            # - Are reporting recent usage as defined by the threhsold value
+            delete_statement = (
+                "DELETE FROM CloudSummaries WHERE "
+                "Month>=%s AND "
```

Comment:
```suggestion
                "Month >= %s AND "
```

would this make it more readable / easier to understand? would also need changing for 2 lines below

Reply:
👍

The spaces aroud the operator seem to be the prefer style for SQL as well :)

Reply:
addressed :)

Reply:
in https://github.com/apel/apel/pull/222/commits/34e2b4f69c6539ed79d4af9ef2d2d29e8c9a8108